cmd/basic/go.mod

```diff
@@ -4,7 +4,7 @@ go 1.19
 
```

Comment:
I recommend making a copy of basic like basic-docker and working from that, leaving basic completely alone.

Reply:
I have created a ```basic-docker``` folder